packages/block-library/src/media-text/edit.js

```diff
@@ -318,11 +318,11 @@ function MediaTextEdit( {
 						onDrag={ imperativeFocalPointPreview }
 					/>
 				) }
-			{ mediaType === 'image' && ( mediaUrl || featuredImageURL ) && (
+			{ mediaType === 'image' && mediaUrl && ! featuredImageURL && (
```

Comment:
Wouldn't it be more reliable to refer to `useFeaturedImage`, similar to [the Cover block](https://github.com/WordPress/gutenberg/blob/ee3fd1a6756b3cd2cd254092c2e879ceb2cc5ec9/packages/block-library/src/cover/edit/inspector-controls.js#L204)?

Reply:
I don't know if it is more reliable but sure lets make them more consistent.